src/App/CommentCard/CommentCard.css

```diff
@@ -0,0 +1,43 @@
+
+.CommentCard {
+  /* primary colors */
+  --primary-color-100: #A4AAA4;
+  --primary-color-200: #639963;
+  --primary-color-300: #2D882D;
+  --primary-color-400: #027702;
+  --primary-color-500: #006600;
+
+  /* secondary colors */
+  --secondary-color-100: #86888A;
+  --secondary-color-200: #54687C;
+  --secondary-color-300: #2A4D6E;
+  --secondary-color-400: #093561;
+  --secondary-color-500: #012A53;
+
+  /* spacing */
+  --spacing-1: 4px;
+  --spacing-2: 8px;
+  --spacing-3: 12px;
+  --spacing-4: 16px;
+  --spacing-5: 20px;
+  --spacing-6: 24px;
+  --spacing-7: 28px;
+  --spacing-8: 32px;
+
+  box-sizing: border-box;
```

Comment:
for `border-box` look to do this in a reset file when working on a larger application.